.circleci/README.md

```diff
@@ -1,22 +1,25 @@
 ## Setting up CircleCI for Scalar DL Node/Web client SDK
 
-### 1. Environment Variables
+### 1.0. Environment Variables
 
 Set environment variables on CircleCi by following [this guide](https://circleci.com/blog/new-on-circleci-import-project-environment-variables/).
 Please set environment variables in CircleCi as follows.
 
-1. Set the environment variables in the [Context](https://circleci.com/gh/organizations/scalar-labs/settings#contexts) with the name `scalar`.
+1.1 Set the environment variables in the [Context](https://circleci.com/gh/organizations/scalar-labs/settings#contexts) with the name `scalar`.
 <pre>
     <b>$DOCKERHUB_USERNAME</b>
     <b>$DOCKERHUB_PASSWORD</b>
 </pre>
 Make sure your DockerHub account has been added by Scalar so that you have the permission to pull scalar-dl image.
 
-2. Set the [Environment Variables](https://circleci.com/blog/new-on-circleci-import-project-environment-variables/) in the project. Click `Add Variable` instead of `Import Variable(s)`
+1.2 Set the [Environment Variables](https://circleci.com/blog/new-on-circleci-import-project-environment-variables/) in the project. Click `Add Variable` instead of `Import Variable(s)`.
 <pre>
     <b>$SCHEMA_CONTENT</b>
 </pre>
-[SCHEMA_CONTENT](https://github.com/scalar-labs/scalardl-node-client-sdk/blob/e119dd687c1c0ed5ee3a3b3d8a945aea70176011/.circleci/config.yml#L55) will be the schema commands that will be executed by `cqlsh`. Here is a [sample schema](https://github.com/pmcfadin/killrvideo-sample-schema/blob/master/killrvideo-schema.cql). 
+[SCHEMA_CONTENT](https://github.com/scalar-labs/scalardl-node-client-sdk/blob/e119dd687c1c0ed5ee3a3b3d8a945aea70176011/.circleci/config.yml#L55) will be the schema commands that will be executed by `cqlsh`. Here is a [sample schema](https://github.com/pmcfadin/killrvideo-sample-schema/blob/master/killrvideo-schema.cql). Make sure the content of `$SCHEMA_CONTENT` has the following criteria :
```

Comment:
I think we don't need a sample schema since it has to be the ledger schema.
Let's simply put the link to the schema. (it won't be visible to unauthorized users though)
```suggestion
[SCHEMA_CONTENT](https://github.com/scalar-labs/scalardl-node-client-sdk/blob/e119dd687c1c0ed5ee3a3b3d8a945aea70176011/.circleci/config.yml#L55) will be the schema that will be applied by `cqlsh`, and it should be a serialized string of [this](https://github.com/scalar-labs/scalar/blob/master/ledger/scripts/create_schema.cql).
```

Reply:
Just a question, is this account is not authorize to view the file at https://github.com/scalar-labs/scalar/blob/master/ledger/scripts/create_schema.cql?

Reply:
Yes, only authorized user can see the file. But I think it's fine since SCHEMA_CONTENT is also be able to be updated by an authorized user.